### PR TITLE
Shout about Add-ons a bit

### PR DESCRIPTION
**Pull request by eyeseast**
- Bigger, darker text for add-ons help link
- Add a New! badge

Screenshot 2023-03-23 at 9 09 25 AM

### CODE REVIEW

**Review by jimkang — COMMENTED** [review]
Hey, I saw a couple of things that might be fine, but should be checked up on.

File: src/langs/json/en.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "emailAddress": "Email address",
     "loading": "Loading...",
     "documentCloud": "DocumentCloud",
-    "home": "Home"
+    "home": "Home",
+    "new": "New"
```
[the remark this answers is not in the document]
Comment (eyeseast) [reply src/langs/json/en.json:7]: How do you switch languages?

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "html-webpack-plugin": "4.5.2",
     "jest": "^24.9.0",
     "mini-css-extract-plugin": "^0.6.0",
-    "node-sass": "^4.13.0",
+    "node-sass": "^4.14.1",
```
[the remark this answers is not in the document]
Comment (eyeseast) [reply package.json:23]: It's not strictly necessary, but it deals with a few of our many audit warnings.

File: src/langs/json/en.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (eyeseast) [reply src/langs/json/en.json:7]: Switched to Spanish and it works. Just falls back to English.

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (jimkang) [reply package.json:23]: A minor version change is supposed to not be a breaking change according to semver, but sometimes dependency updates have unpredictable cascading effects. So, I'm cool with this, but just noting so we have something to look at in the case something bizarre happens later.

**Pull request merged**